wire/core/Pageimage.php

```diff
@@ -1810,6 +1897,120 @@ protected function ___install($filename) {
 		}
 	}
 
+	
+	/**
+	 * Get verbose DebugInfo, optionally with individual options array, @horst
+	 * (without invoking the magic debug)
+	 * 
+	 * @return mixed: array | string
```

Comment:
One more note about return format: technically this means that the return value type is `mixed:`, which is not a valid type. `:` should be removed or separated from `mixed` with a space, though if there are only two possible return types I'd recommend `@return array|string` instead.

Reply:
In addition note that parameters (`$options` and `$returnAsString`) are currently not mentioned here.